Header header

# x/y location, in pixels
geometry_msgs/Vector3 point

# vector from sensor frame; LaserDotFinderNode provides this as a unit vector, which may be projected by VisualAltimeterNode
geometry_msgs/Vector3 vector

# color
ColorRGBA color
